src/rust/ensogl/src/display/scene.rs

```diff
@@ -779,20 +779,21 @@ impl SceneData {
 
         let display_object = display::object::Instance::new(&logger);
         let context        = web::get_webgl2_context(&dom.layers.canvas);
-        let sub_logger     = logger.sub("shape_dirty");
+        let sub_logger     = Logger::sub(&logger,"shape_dirty").into();
         let shape_dirty    = ShapeDirty::new(sub_logger,Box::new(on_mut.clone()));
-        let sub_logger     = logger.sub("symbols_dirty");
+        let sub_logger     = Logger::sub(&logger,"symbols_dirty").into();
```

Comment:
We should get rid of Logger::sub(…).`into` calls in this function and rather have the `new` function accept `Into<Logger>`